UnityProject/Assets/Scenes/Confetti.unity

```diff
@@ -0,0 +1,292 @@
+%YAML 1.1
```

Comment:
should we be committing this scene file? as far as I can tell it is not used.

Reply:
It's not used in-game, but it's what I was using to efficiently prototype in the editor without the UI getting in the way. Probably useful to keep if we iterate on it more.